modules/ROOT/pages/mule-app-dev.adoc

```diff
@@ -0,0 +1,71 @@
+= Mule Application Development
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Mule applications enable you to perform data integrations for your use cases.
+Typical Mule applications read data from internal and external sources,
+process and transform data to the required formats and structures, and
+write that output to the systems and servers where you store and use the
+transformed data.
+
+//TITLE TBD:
+== Mule Applications
+
+You can start a Mule app with or without an API specification, connect to
```

Comment:
Should we be more explicit about the different use cases for building mule applications?. There a very well defined set of use cases like implementing an API, create a new API by orchestrating other APIs, data integration, batch processing, b2b integration And there are other use cases that can also be implemented with the Mule Runtime with some restrictions like process automation, ETL, etc. This section seems key and we should engage with PMs.